omod/src/main/webapp/tags/conceptField.tag

```diff
@@ -76,6 +77,10 @@
 		
 	})
 	
+	function convert(conc){
```

Comment:
What is this function doing?

Reply:
it converts the  string(concept object) eg "concept #12"  to a numeric string (concept id) eg "12"